style.css

```diff
@@ -1,37 +1,83 @@
-/* * är en universal selector som återställer elements så de har 0 margins, så de ser likadant ut i alla browsers*/
 *{
-    margin: 0;
+  margin: 0;
+  }
+
+  body, html {
+  height: 100%;
   }
+
   
-/******* BAKGRUNDSBILD ******/ 
-/* Byt ut bilden 
-/* Linear gradient gör så att bilden blir mörkare och texten blir tydligare. detta kan du ta bort/ändra om du vill
-/* lägg till en storlek på bilden ex. "height: 500px" 
-/* Se om du kan få bilden att täcka toppen av sidan hint -> background-repeat: no-repeat;*/
-/* lägg till position: relative; så hamnar bilden under texten. */  
+  /* The image have to be darker so the white text is easier to see. */
+  /* Background-position left is for the responsive style, the cat is in the left of the picture and I want her to be visible even if I see this on a mobile screen. */
 
 .hero-content {
-  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/apelsiner.png");
-  height: 500px;
-}
-  
-  /****** INFORMATION OVANPÅ BILDEN*****/
-  /* Lägg till position: absolute; Då lägger sig texten ovanpå bilden 
-  /* positionera texten rätt genom att använda tillexempel top, right, left, bottom.
-  /* eller positionera texten i center med hjälp av tillexmepel flexbox */
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/MittensTheKhajiit.jpg");
+  height: 70%;
+  background-position: left;  
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  }
+
+  /* Position absolute puts text upon the picture, and top and left places the text in the middle. */
+  /* Transform will reposition the text to the middle of the picture if the screen is small, in the mobile I want the cat in the center and the text upon it. */
 
 .hero-text{
-    color:white;
+  font-family: 'Gloria Hallelujah', cursive;
+  color:white;
+  text-align: center;
+  position: absolute; 
+  top: 50%;  
+  left: 50%; 
+  transform: translate(-50%, -50%); 
+}
+
+.hero-text h1 {
+  font-size: 60px;
+}
+
+.hero-text p {
+  font-size: 30px;
+  padding-bottom: 20px;
 }
 
-    /****** STYLA DIN KNAPP HÄR NEDAN****/
-    /*Här hittar du lite styling tips till din knapp https://www.w3schools.com/css/css3_buttons.asp */
  
 .btn{
-  color: white;
+  font-family: 'Open Sans', sans-serif; 
+  font-size: 15px;
+  outline: 0;
+  border: none;
+  padding: 10px;  
+  color: black;
+  font-weight: bold;
+  background-color: #b2ebf2; 
+  text-align: center;
+  cursor: pointer;
+}
+
+.btn:hover{
+  background-color:	#e0f7fa;
+}
+
+
+.page-content p {
+  padding: 100px;
+  font-family: 'Open Sans', sans-serif; 
 }
 
-/****** Lägg till en extra bild här nedan om du vill.****/
-/*Glöm inte att lägga till ett class namn på din image tag */
\ No newline at end of file
+/*Responsive styles*/
+
+@media screen and (max-width: 500px) {
+  .hero-text h1 {
+    font-size: 40px;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .hero-text p {
+    font-size: 20px;
+  }
+}
```

Comment:
Extra  ⭐️for adding media queries. Is it enough that you only apply the font-size changes for both elements under only one media query only. I suggest you apply the following:

 @media screen and (max-width: 500px) {
    .hero-text h1 {
        font-size: 40px;
      }
    .hero-text p {
        font-size: 20px;
      }
  }

Reply:
Thank you for all your feedback! <3  And I'm glad that you liked the character =) 
Have a nice week!